connectors/opa/connector.go

```diff
@@ -30,12 +31,26 @@ type ConnectorController struct {
 	Log          zerolog.Logger
 }
 
-func NewConnectorController(opaServerURL string) *ConnectorController {
+func NewConnectorController(opaServerURL string) (*ConnectorController, error) {
+	log := logging.LogInit(logging.CONNECTOR, "opa-connector")
+	retryClient := retryablehttp.NewClient()
+	if strings.HasPrefix(opaServerURL, "https") {
+		config, err := fybrikTLS.GetClientTLSConfig(&log)
+		if err != nil {
+			log.Error().Err(err)
+			return nil, err
+		}
+		if config != nil {
```

Comment:
and if we work with a default config and config == nil, we don't have to create TLS?